src/components/include/test/application_manager/mock_state_controller.h

```diff
@@ -113,6 +113,7 @@ class MockStateController : public am::StateController {
   MOCK_METHOD2(DeactivateApp,
                void(am::ApplicationSharedPtr app,
                     const am::WindowID window_id));
+  MOCK_METHOD1(OnTimeOutActivateAppRequest, void(uint32_t hmi_app_id));
```

Comment:
@VladSemenyuk I don't see such function in StateController. Is this something what should be removed?

Reply:
Done